students/FlorentinPopescu/07_Mailroom_Part1_L3_FP.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jan 14 22:01:43 2019
+@author: Florentin Popescu
+"""
+
+#===================LESSON_03====================
+# Mailroom, Part 1-------------------------------
+#================================================
+
+#================================================
+# Sending a Thank You note
+#================================================
+import re
+
+#donors on file
+donors = [['Name1', 1, 10, 100], 
+          ['Name2', 5.50, 15.50, 25.50],
+          ['Name3', 1000000],
+          ['Name5', 10.50, 20.75, 4.25],
+          ['Name6', 999.99, 99.99, 9.99],
+          ['Name7', 0.75,  0.25]]
+
+
+choices = ["Send a Thank You", "Create a Report", "Quit"]
+
+#-----------------------------------------------
+def donor_list(table):
+    return [item[0] for item in table]
+
+#-----------------------------------------------
+def add_donor(table, item):
+    return table.append([item])
+
+#-----------------------------------------------
+def check_name(string):
+    '''
+    Test if donor's name contains special characters
+    '''
+    regex = re.compile("[-+=@_!#$%^&*()<>?/\|}{~:]")
+    if(regex.search(string) == None): 

```

Comment:
needs space, no parens and use `is` for object checking like `None`.. or use more pythonic checking:
```
if not regex.search(string):
```
```
return bool(regex.search(string))
```